src/Bicep.Core.UnitTests/FileSystem/FileResolverTests.cs

```diff
@@ -203,24 +213,40 @@ public void TryReadAtMostNCharacters_RegardlessFileContentLength_ReturnsAtMostNC
 
         [TestMethod]
         public void In_memory_file_resolver_should_simulate_directory_paths_correctly()
-        {   //file://path
+        {
             var fileTextsByUri = new Dictionary<Uri, string>
             {
-                [new Uri("file://path/to/file.bicep")] = "param foo int",
-                [new Uri("file://path/to/nested/file.bicep")] = "param bar int",
-                [new Uri("file://path/toOther/file.bicep")] = "param foo string"
```

Comment:
There's no circumstance under which a user might be supplying a file URI like these, is there (e.g., `module mod 'file://...' = {` or the like)? These look like errors in the original test, but wanted to make sure this won't change behavior in a way users would notice.

Reply:
Correct - we don't support schemes in file paths (or even absolute paths).